srcpkgs/NetworkManager-openconnect/template

```diff
@@ -1,17 +1,21 @@
 # Template file for 'NetworkManager-openconnect'
 pkgname=NetworkManager-openconnect
-version=1.2.6
-revision=4
+version=1.2.8
+revision=1
 build_style=gnu-configure
-configure_args="--disable-static"
+configure_args="--disable-static --with-gtk4"
 hostmakedepends="pkg-config intltool"
-makedepends="gcr-devel gtk+3-devel liblz4-devel libnma-devel
- openssl-devel libsecret-devel libxml2-devel openconnect-devel"
-depends="openconnect"
+makedepends="gcr-devel gtk4-devel liblz4-devel libnma-devel
+ openssl-devel libsecret-devel libxml2-devel openconnect-devel webkit2gtk-devel"
```

Comment:
webkit2gtk uses gtk3, this program gtk4 and mixing them does not work. You might want to wait for #39488 and then use libwebkit2gtk50

Reply:
Strange enough, it works locally. Building with gtk3 produces build erorrs.

Reply:
hmm, then I guess they might be using gtk3 webkit and gtk4 for different things and not mix them